### ISSUE TITLE
Jira plugin isn't working anymore with basic auth

### ISSUE THREAD

**Issue by angystardust** [psss/did#163]
Hi, I've just upgraded my _did_ setup to 0.11 (did-0.11-1.fc29.noarch) using the COPR repository but I have to revert back to _did-0.9_ since jira plugin with basic auth is not working anymore.
It seems that now _did_ tries to do a gss auth even if you specify to use the basic auth in the config file:

```
$ did yesterday
Status report for yesterday (2018-12-02 to 2018-12-02).

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
My User <myuser@example.com>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ERROR  401 Client Error:  for url: https://jira.eidosmedia.com/rest/auth/latest/session
 ERROR  Jira authentication failed. Try kinit.
```

My config:

```
[jira-mycompany]
type = jira
prefix = MYGROUP
project = MYGROUP
url = https://jira.example.com/
auth_url = https://jira.example.com/rest/auth/latest/session
auth_type = basic
auth_username = myuser
auth_password = mypassword
```

**Comment by geoand** [comment]
Saw the same exact thing :)

**Comment by psss** [comment]
Thanks for reporting the bug. This should be fixed by 99280c0.

**Comment by angystardust** [comment]
Thank you so much for fixing so fast! :100:
